kafka/api.html

```diff
@@ -672,11 +677,26 @@ <h2 class="toggleButton" title="Click to hide Index section">
           func (c *Consumer) GetMetadata(topic *string, allTopics bool, timeoutMs int) (*Metadata, error)
          </a>
         </dd>
+        <dd>
+         <a href="#Consumer.GetRebalanceProtocol">
+          func (c *Consumer) GetRebalanceProtocol() string
+         </a>
```

Comment:
not much point exposing this imo, and slightly better not to, but doesn't matter.